docs/source/api/apollo-gateway.mdx

```diff
@@ -67,35 +80,48 @@ The core of the federated gateway implementation. For an example, see the ["impl
 
   * `debug`: `Boolean`
 
-    With debug enabled, the server will log startup messages as well as query plans during incoming requests.
+    If `true`, the gateway logs startup messages, along with the query plan for
+    each incoming request. The default value is `false`.
 
 #### Returns
 
-An `ApolloGateway` instance, which is intended to be passed as the `gateway` configuration option of `ApolloServer` like so:
+An `ApolloGateway` instance, which you then pass as the `gateway` configuration option to the `ApolloServer` constructor, like so:
 
 ```javascript
 const server = new ApolloServer({
   gateway: new ApolloGateway({ serviceList: [...] }),
 });
 ```
 
-## RemoteGraphQLDataSource
+## `RemoteGraphQLDataSource`
 
-The `RemoteGraphQLDataSource` export is used to connect to underlying federated services from the gateway. It allows for customizations of its request using the `willSendRequest` function. By default, the `ApolloGateway` creates a new `RemoteGraphQLDataSource` for each service using the `url` from the service definition as its target location.
+A `RemoteGraphQLDataSource` object represents a connection between your
+federated gateway and a single implementing service. You can customize this
+communication by defining a `willSendRequest` function that lets you modify each
```

Comment:
This is a miss in my own updates, but we should either remove the nod to `willSendRequest` or also nod to `didReceiveResponse` now that it's documented.